controller/src/main/java/io/pravega/controller/store/stream/PravegaTablesStreamMetadataStore.java

```diff
@@ -131,6 +134,19 @@ public OperationContext createStreamContext(String scopeName, String streamName,
         return new StreamOperationContext(scope, stream, requestId);
     }
 
+    @Override
+    public CompletableFuture<Void> addEntryToDeletingScope(String scope, OperationContext context, ScheduledExecutorService executor) {
+        return Futures.toVoid(createTableIfAbsent(() -> storeHelper.addNewEntry(
+                DELETING_SCOPES_TABLE, scope, UUID.randomUUID(), UUID_TO_BYTES_FUNCTION, context.getRequestId()), context)
```

Comment:
I had mentioned this earlier as well, the UUID added to this table should be UUID of the scope and not a random UUID. I don't see the point in adding a random UUID here.

Reply:
It may be better to rewrite `addEntryToDeletingScope`  like this:
public CompletableFuture<Void> addEntryToDeletingScope(String scope, OperationContext context, ScheduledExecutorService executor) {
Futures.completeOn((PravegaTablesScope) getScope(scope, context)).addEntryToDeletingScope());
}

Inside PravegaTablesScope class `withCreateTableIfAbsent` can be reused.